lexrank/tools/assemble_stopwords.py

```diff
@@ -23,11 +22,29 @@ def assemble_stopwords():
                 lang_words.add(clean_text(name))
 
         stopwords_map[lang] = list(lang_words)
-
         sys.stdout.write('  ' + lang + ' ' + 'stopwords collected\n')
 
+    if not stopwords_map:
+        return
+
+    out_file = settings.ASSETS_ROOT / 'stopwords.json.gz'
+
     with gzip.open(out_file, mode='wt', encoding='utf-8') as fp:
         json.dump(stopwords_map, fp)
 
         msg = 'Stopwords written to {out_file}\n'.format(out_file=out_file)
         sys.stdout.write(msg)
```

Comment:
let us think about logger, create issue for future

Reply:
Done